app/routes/settings.js

```diff
@@ -21,6 +21,36 @@ export default TravisRoute.extend({
     return repo.get('envVars.promise');
   },
 
+  fetchCronJobs() {
+    var repo = this.modelFor('repo');
+    var apiEndpoint = config.apiEndpoint;
+
+    return $.ajax(apiEndpoint + "/v3/repo/" + repo.get('id'), {
```

Comment:
Elsewhere I’ve seen an `ajax` service used for Ajax calls, do you know whether that should be used here? I believe it’s used because of browser incompatibilities, though I don’t know much more than that.